app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -19,8 +19,12 @@
   padding: 26px;
   position: relative;
   border-radius: 10px;
-  color: #CCC;
+  color: #999;
   font-size: 1em;
+
+  &:hover {
+    color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.